src/files/infra/repositories/FilesRepository.ts

```diff
@@ -23,6 +23,16 @@ export interface GetFilesQueryParams {
   searchText?: string;
 }
 
+export interface GetFilesTotalDownloadSizeQueryParams {
+  includeDeaccessioned: boolean;
+  mode?: string;
+  contentType?: string;
+  accessStatus?: string;
+  categoryName?: string;
+  tabularTagName?: string;
+  searchText?: string;
+}
+
```

Comment:
You could have a common interface with the search criteria properties and then extend it with the other properties to not repeat the same properties in both interfaces when we already know those properties are the search criteria

```
export interface GetFilesQueryParams extends SearchCriteriaQueryParams {
  includeDeaccessioned: boolean;
  limit?: number;
  offset?: number;
  orderCriteria?: string;
}

export interface GetFilesTotalDownloadSizeQueryParams extends SearchCriteriaQueryParams {
  includeDeaccessioned: boolean;
  mode?: string;
}
```

Reply:
Done!